### PR TITLE
Moved cortical borers into a modpack.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
Also added some symbiote job slot stuff from an old Bay PR.

Borers in general are quite under-maintained, I'm going to rewrite them soon I think, this PR is to make that easier.

### CODE REVIEW

File: mods/borers/mob/overrides.dm [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#define HAS_BRAIN_WORMS(HOST) locate(/mob/living/simple_animal/borer) in HOST.contents
```
Comment (Techhead0) [inline mods/borers/mob/overrides.dm]: Wrap in (), just in case, due to the `in` operator's low precedence

File: code/_helpers/medical_scans.dm [hunk 2]
```diff
@@ -70,6 +69,7 @@
 		O["brute_dam"] = E.brute_dam
 		O["burn_dam"] = E.burn_dam
 		O["scan_results"] = E.get_scan_results(tag)
+		O["tumors"] = E.has_growths()
```
Comment (Techhead0) [inline code/_helpers/medical_scans.dm:72]: Looking forward to seeing Chinsky add Space Cancer

Comment (comma) [reply code/_helpers/medical_scans.dm:72]: It's been on my todo for ages!

File: mods/psionics/system/psionics/mob/borer_power.dm [hunk 3]
```diff
@@ -2,6 +2,11 @@
 	var/image/aura_image
```
Comment (comma) [inline mods/psionics/system/psionics/mob/borer_power.dm:3]: Maybe this file should be included if borer mod is included only

Comment (MistakeNot4892) [reply mods/psionics/system/psionics/mob/borer_power.dm:3]: I can't see a way to reliably do that since the include order is not predictable. :(

**Review by comma — APPROVED** [review]

**Pull request merged**